package/autoyast2.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Apr  5 08:41:25 UTC 2021 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Add the 'mkfs_options' element to the schema (bsc#1184268).
```

Comment:
no version bump is intetional, right?

Reply:
Yes, it is. I plan to release a few fixes together.